bundle/manifests/backstage-operator.clusterserviceversion.yaml

```
@@ -206,7 +206,7 @@ spec:
                   value: quay.io/fedora/postgresql-15:latest
                 - name: RELATED_IMAGE_backstage
                   value: quay.io/janus-idp/backstage-showcase:next
-                image: quay.io/janus/operator:next
+                image: quay.io/janus/operator:v0.0.1
```

This tag does not exist.

```suggestion
                image: quay.io/janus/operator:next
```

---

Good. Done.